railties/test/railties/engine_test.rb

```diff
@@ -252,6 +254,64 @@ def index
       assert_equal "Hello bukkits\n", response[2].body
     end
 
+    test "does not eager load controller actions in development" do
```

Comment:
Those tests seems to be in the wrong place. Should not this be in the application/configuration_test? While engines defines initializers they don't boot the application so eager load tests should not be here.

https://github.com/rails/rails/blob/master/railties/test/application/configuration_test.rb